### DISCUSSION TITLE
Question about win 11 taskbar

### DISCUSSION THREAD

**Discussion by ghost** [valinet/ExplorerPatcher#562] [General]
Hi valinet, i was wondering this since you know about registry edit. Would it be possible with the current windows 11 taskbar to remove the gap/space between taskbar and previewing apps?

![save2](https://user-images.githubusercontent.com/96125502/146010811-3325cf35-3010-4f99-a8af-6eb2cae4b3ee.png)

**Comment by valinet** [comment]
I don’t know, sorry.

**Reply by valinet** [reply]
I don’t know.

**Reply by valinet** [reply]
Anyway, best way to make Windows 11 exactly like Windows 10 is to install Windows 10.
